src/lib.rs

```diff
@@ -300,41 +346,44 @@ mod tests {
         ));
     }
 
-    #[test]
-    fn test_if() {
-        let program = parse(
-            "
-            function x() {
-                if (1 == 1) {
-                    return 42;
-                } else {
-                    return 0;
-                }
-            }
-            ",
-        );
-
-        let mut program = program.unwrap();
-        assert_eq!(program.nodes.len(), 1);
-
-        let mut function = match program.nodes.pop().unwrap() {
-            ast::RootNode::Function(function) => function,
-            _ => panic!("Expected function"),
-        };
-
-        assert_eq!(function.statements.len(), 1);
-
-        let if_ = match function.statements.pop().unwrap() {
-            ast::Statement::If(if_) => if_,
-            _ => panic!("Expected if"),
-        };
-
-        assert!(
-            matches!(if_.condition, ast::Expression::Equal(n, m) if *n == ast::Expression::Primitive(ast::Primitive::Number(1.0)) && *m == ast::Expression::Primitive(ast::Primitive::Number(1.0)))
-        );
-        assert_eq!(if_.then_statements.len(), 1);
-        assert_eq!(if_.else_statements.len(), 1);
-    }
+    // #[test]
```

Comment:
Any reason this is commented out? Are we no longer supporting fns at the root, or was that never supported?

Reply:
I added them during the initial rewrite of the parser from Javascript, but I don't think we support them in the indexer, so I thought it's better to disallow them for now.

Reply:
We can specifically add them back in at some point then, np leaving them out for now!